src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminCourtAddressControllerTest.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmcts.dts.fact.controllers.admin;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.dts.fact.exception.NotFoundException;
+import uk.gov.hmcts.dts.fact.model.admin.CourtAddress;
+import uk.gov.hmcts.dts.fact.services.admin.AdminCourtAddressService;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(AdminCourtAddressController.class)
+@AutoConfigureMockMvc(addFilters = false)
+public class AdminCourtAddressControllerTest {
+    private static final String TEST_SLUG = "court-slug";
+    private static final String BASE_PATH = "/admin/courts/";
+    private static final String ADDRESSES_PATH = "/" + "addresses";
```

Comment:
Super minor again, string format if you want. Think the reason is because of readability and/or performance/bug reduction - up to you though!

Reply:
I think concatenation is alright in this case as per previous comment